coreos/provision-coreos.sh

```diff
@@ -45,9 +45,13 @@ chmod +x docker-compose
 mkdir -p postgres
 ./docker-compose build
 
-sudo cp nanocloud.architecture.service /etc/systemd/system/nanocloud.architecture.service
+docker pull glyptodon/guacd
```

Comment:
Currently community has been tested with guacd 0.9.8. We should stick to this version.
Regarding ambassador its seems like 0.9.1 refers to latest. As we already know it works I suggest we stick to this version.

If you agree, can you please update your commit accordingly ?